example/helloWorld/src/helloworld.cpp

```diff
@@ -0,0 +1,190 @@
+/**
+ * \file
+ * Copyright 2014-2015 Benjamin Worpitz
+ *
+ * This file is part of alpaka.
+ *
+ * alpaka is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * alpaka is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with alpaka.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ * \author Erik Zenker
+ *
+ */
+// Alpaka
+#include <alpaka/alpaka.hpp>
+
+// STL
+#include <iostream>
+
+
+/**
+ * Helper functions that calculates the global thread id depending
+ * on the number of threads set by the workdiv.
+ */
+template <typename T_Acc>
+size_t globalThreadIdx(T_Acc const &acc){
```

Comment:
Could you rename this function to `linearizedGlobalThreadIndex` because what you call `threadIdx` inside is already the real  `globalThreadIdx` that is identical to what I call `gridThreadIdx`.